index.html

```diff
@@ -8,9 +8,9 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
+    <div class="container">
         <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+            <p class="heading">My logo</p>
         </div>
         <div>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer,  I think it would be better to replace the ```<div>`` tag with a ```<section>``` tag, As this will make your codes cleaner and professional 🙏 

You may follow this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info

Reply:
- [ ] Kindly as suggested by the previous reviewer,  I think it would be better to replace the ```<div>`` tag with a ```<section>``` tag, As this will make your codes cleaner and professional 🙏 

You may follow this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info